### ISSUE TITLE
Hide empty side-bar if no custom sidebar is defined

### ISSUE THREAD

**Issue by edbarnard** [ScopeFoundry/ScopeFoundry#5]
Currently a blank area when no custom sidebar is defined:

<img width="283" alt="screen shot 2017-01-19 at 10 12 02 am" src="https://cloud.githubusercontent.com/assets/6744917/22119262/be451e74-de2f-11e6-970f-542bb3168b08.png">

**Comment by alanrbuckley** [comment]
Copied file "df_sidepanel.ui" to local working folder and included the following in RandomGenApp.py:
```python
def setup(self):
        self.side_ui_filename = sibling_path(__file__, "df_sidepanel.ui")
        self.side_ui = load_qt_ui_file(self.side_ui_filename)
```
Also added conditional which successfully loads sidepanel. 

I have yet to figure out how to collapse the QWidget section, more to come next week.
```python
        if self.side_ui is not None:
            self.add_quickbar(self.side_ui)
        else:
            #Collapse sidebar
            pass
```

**Comment by alanrbuckley** [comment]
Commit:  a8f10fb16af87e1b2c4181a64c1821de0df82383 -> Controllers branch

```python
        # Uncomment below line to test whether quickaccess_scrollArea hides correctly.
        # self.side_ui = None    
            
        if self.side_ui is not None:
            self.add_quickbar(self.side_ui)
            # Inject side_ui into sidebar of self.ui. Done.
            pass
        else:
            # Collapse sidebar
            self.ui.quickaccess_scrollArea.setVisible(False)
            pass
        
```
Modified base_app.py to hide scrollarea if no "df_sidepanel.ui" file is found.

**Comment by edbarnard** [comment]
Fixed and merged in 80d57dc32c70bb4c7ce77ac4cd196b78486d4b02
